app/templates/components/public/featured-speaker-list.hbs

```diff
@@ -0,0 +1,8 @@
+<h3>{{t 'Featured Speakers'}}</h3>
+<div class="speaker-list ui grid">
+  {{#each speakers as |speaker|}}
+    {{#if speaker.featured}}
```

Comment:
Yes. So, you need a computed property named featuredSpeakers right ? Instead of two arrays ?